README.md

```diff
@@ -129,13 +129,34 @@ The architecture with the components involved is documented
 
 To trigger e2e test on a PR, use the following phrases:
 
-### integration tests
+### Integration tests
 
 - **/test-ubuntu-e2e-integration-main** runs integration e2e tests with CAPM3
   API version v1beta1 and branch main on Ubuntu
 - **/test-centos-e2e-integration-main** runs integration e2e tests with CAPM3
   API version v1beta1 and branch main on CentOS
 
+Release-1.4 branch:
+
+- **/test-ubuntu-e2e-integration-release-1-4** runs integration e2e tests with CAPM3
+  API version v1beta1 and branch main on Ubuntu
```

Comment:
The explanation here and test-ubuntu-e2e-integration-release-1-3 are same. It would be better to add capm3 release-1.4 or release-1.3 in the text description